bitnami/moodle/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: moodle
 sources:
   - https://github.com/bitnami/bitnami-docker-moodle
   - https://www.moodle.org/
-version: 12.0.3
+version: 12.0.4
```

Comment:
Actually, could you increase the minor here? Since this is adding a new feature.